### DISCUSSION TITLE
MSSQL default tag crashes on GitHub runners

### DISCUSSION THREAD

**Discussion by ecki** [testcontainers/testcontainers-java#9280] [General]
Since a random unrelated change in a GitHub project my Maven integration tests which use Testcontainer fail. It looks like the [MSSQLServerContainer 2017-CU12](https://github.com/testcontainers/testcontainers-java/blob/751aa4eee75aa135d365647d8b0b32347f98151d/modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLServerContainer.java#L22) default tag stopped working and it looks like it’s a crash.

Anybody experienced this? Has the tag changed? This container works fine in my local Docker Desktop with WSL2 tests. Here is the first failed test: https://github.com/seeburger-ag/bis-resources/actions/runs/10995873923 ([crash log](https://github.com/seeburger-ag/bis-resources/actions/runs/10995873923/job/30527740536#step:4:307))

After switching this project to 2022-latest, it continued to work.

BTW: will that default tag be bumped?

**Comment by eddumelendez** [comment]
Default constructor is deprecated and not recommended anymore. You can use 

```java
MSSQLServerContainer<?> mssqlServerContainer = new MSSQLServerContainer<>(
                DockerImageName.parse("mcr.microsoft.com/mssql/server:2022-CU14-ubuntu-22.04")
```

**Reply by ecki** [reply]
I know I can (and I do), but I would expect the default tracks latest releases anyway

**Reply by eddumelendez** [reply]
No, that's not the case. Updating the default image could break tests by just updating the library. Thats the reason why it has been deprecated and users should set the version they are using.

**Comment by ecki** [comment]
Yeah that’s one way of seeing it, the other is if I don’t specify a version I don’t care if it changes (in fact I expect the library to maintain a recent version). But yeah I guess whatever, this discussion is about another problem, I am curious if anybody knows why the old version does not work anymore on GitHub?

**Reply by eddumelendez** [reply]
You can follow this issue https://github.com/actions/runner-images/issues/10649

**Comment by ecki** [comment]
Good find, thanks!
